tools/CMakeLists.txt

```diff
@@ -9,3 +9,15 @@ endif()
 if(LIBRW_PLATFORM_PS2)
     add_subdirectory(ps2test)
 endif()
+
+if(UNIX)
```

Comment:
This means the samples aren't built on Windows.
Is there a problem/error?

Reply:
In CI check, I met build error at tools/playground on Windows.
But I can not test.because I have no Win machine so I excluded. :-)

Reply:
This one?
https://github.com/aap/librw/runs/2199399063?check_suite_focus=true
I'll look into it

Reply:
The reason for the error is that executable targets linked to the librw skeleton are gui apps and need to be linked with `/subsystem:WINDOWS` (`WIN32` on cmake).

This (and some extra changes) makes it build on my system + CI:
my changes on top of this pr: https://github.com/madebr/librw/tree/pr69
ci results: https://github.com/madebr/librw/actions/runs/691478341

complete diff: https://github.com/madebr/librw/compare/21808166529b10e9fdd6680075e43ca871390fd9..pr69

Reply:
@seoyoungjin
Can you check my last comment?